lib/services/plan_payment_service.ts

```diff
@@ -0,0 +1,23 @@
+import { Stripe } from 'stripe';
+
+export interface CreatePlanPaymentRequestBody {
+  /** The promotion codes for coupon */
+  couponCode?: string;
+
+  /** The payment method on stripe */
+  stripePaymentMethodId: string;
+
+  /** The ID of the application. */
+  appId?: string;
+
+  /** The check option plan for user by year or month. */
+  isYearlyPayment?: boolean;
+
+  /** The ID of list-plan. */
+  selectedPlanId: string;
+}
+
+export type PlanPaymentResponse = Stripe.Response<Stripe.Subscription> & {
```

Comment:
@loan9202 
I missed this one, please be careful with the naming.
```suggestion
export type CreatePlanPaymentResponse = Stripe.Response<Stripe.Subscription> & {
```

Reply:
I updated